File: ruby/fizzbuzz.rb
  result = []

  (1...16).each do |number|
    if number % 3 == 0 && number % 5 == 0
      result << "FizzBuzz"
    elsif number % 5 == 0
      result << "Buzz"
    elsif number % 3 == 0
      result << "Fizz"
    else
      result << number
    end
  end

  puts result

File: python/fizzbuzz.py
result = []

for number in range(40):
    if number % 3 == 0 and number % 5 == 0:
        result.append("FizzBuzz")
    elif number % 5 == 0:
        result.append("Buzz")
    elif number % 3 == 0:
        result.append("Fizz")
    else:
        result.append(number)

print(result)
